biletmajster-backend/Controllers/EventApi.cs

```diff
@@ -143,6 +149,121 @@ public virtual async Task<IActionResult> CancelEvent([FromRoute][Required] strin
             return StatusCode(404, new ErrorResponse { Message = "Event not found" });
         }
 
+        /// <summary>
+        /// Cancel event
+        /// </summary>
+        /// <param name="id">id of Event</param>
+        /// <param name="path">path of photo</param>
+        /// <response code="204">deleted</response>
+        /// <response code="403">invalid session</response>
+        /// <response code="404">id or path not found</response>
+        [HttpDelete]
+        [Route("/events/{id}/photos")]
+        [Authorize]
+        [ValidateModelState]
+        [SwaggerOperation("DeletePhoto")]
+        public virtual async Task<IActionResult> DeletePhoto([FromRoute][Required] string id, [FromHeader][Required()] string path)
+        {
+            var @event = await _modelEventRepository.GetEventByIdAsync(long.Parse(id));
+            if (@event == null)
+            {
+                _logger.LogDebug($"Event with id {id} not found");
+                return StatusCode(404, new ErrorResponse { Message = "Event not found" });
+            }
+            var photo = @event.EventPhotos.Find(x => x.DownloadLink == path);
+            if (photo == null)
+            {
+                _logger.LogDebug($"Can not find image with link: [{path}] belong to event with id:{id}");
+                return StatusCode(404, new ErrorResponse { Message = "Path not found" });
+            }
+            @event.EventPhotos.Remove(photo);
+            if (!await _eventPhotosRepository.DeletePhoto(photo))
+            {
+                return StatusCode(403, new ErrorResponse { Message = "invalid session" });
+            }
+            if (!await _modelEventRepository.UpdateEvent(@event))
+            {
+                return StatusCode(403, new ErrorResponse { Message = "invalid session" });
+            }
+            return StatusCode(204);
```

Comment:
Can we also delete the photo from our Blob Storage? (would be nice, but if the names are random (see next comment) then we don't need to do this)